drivers/input/keyboard/miyoo_kbd.c

```diff
@@ -592,6 +608,35 @@ static void scan_handler(unsigned long unused)
       val|= MY_R3;
       hotkey_actioned = true;
 	}
+  } else if(miyoo_ver == 4) {
+    if((val & MY_R) && (val & MY_TA)) {
+      if(!hotkey_down) {
+      static char * shutdown8_argv[] = { "/bin/sh", "-c", "/bin/kill -9 $(/bin/ps -al | /bin/grep \"/mnt/\" | /bin/grep -v \"/kernel/\" | /usr/bin/tr -s [:blank:] | /usr/bin/cut -d \" \" -f 2) ; /bin/sleep 0.1 ; /bin/sync ; /bin/swapoff -a ; /sbin/poweroff",  NULL };
```

Comment:
I would recommend to bind this special hotkey to just ``/bin/kill -9``, it would be more usefull for normal user (especially for those apps which don't have working SDL_Quit or exit(0) logic). Moreover that what we are planning for upcoming 2.0 release and is set on default miyoo_layout.

Reply:
This is used too
```
static char * shutdown90_argv[] = { "/bin/sh", "-c", "/bin/kill -9 $(/bin/ps -al | /bin/grep \"/mnt/\")" , NULL };
```
I think this may be worth